src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -528,6 +531,19 @@ struct amdxdna_drm_set_state {
 	__u64 buffer; /* in */
 };
 
+
+/**
+ * struct amdxdna_drm_syncobjs - Signal or wait on array of DRM timelined sync objects.
+ * @handles: Array of handles of sync objects.
+ * @points: Array of time points for each sync objects.
```

Comment:
@count: ?

Reply:
Oh, right. I forgot to add comment for "count" argument. I'll add it.